### PR TITLE
Updated en-docs about page transitions

### PR DESCRIPTION
**Pull request by samuells**
- change `transition` key to `pageTransition`
- added info from which version and when `transition` will be deprecated
- added subsection about transition mode

### CODE REVIEW

File: en/api/pages-transition.md [hunk 1]
```diff
@@ -1,34 +1,36 @@
 ---
-title: "API: The transition Property"
+title: "API: The pageTransition Property"
 description: Nuxt.js uses the `<transition>` component to let you create amazing transitions/animations between your pages.
 ---
 
 # The transition Property
```
Comment (manniL) [inline en/api/pages-transition.md]:
```suggestion
# The pageTransition Property
```

File: en/api/pages-transition.md [hunk 2]
```diff
@@ -86,13 +88,26 @@ You can also define methods in the `transition`, these are for the [JavaScript h
 
 *Note: it’s also a good idea to explicitly add `css: false` for JavaScript-only transitions so that Vue can skip the CSS detection. This also prevents CSS rules from accidentally interfering with the transition.*
 
+### Transition Mode
+
+**Default transition mode for pages differs from the default mode in VueJS**. The `pageTransition` mode is by default set to `out-in`. If you want to run leaving and entering transitions simultaneously, you have to set the mode to empty string `mode:""`.
```
Comment (manniL) [inline en/api/pages-transition.md]:
```suggestion
**The default transition mode for pages differs from the default mode in Vue.js**. The `pageTransition` mode is by default set to `out-in`. If you want to run leaving and entering transitions simultaneously, you have to set the mode to the empty string `mode: ''`. 
```

**Comment by samuells** [comment]
@manniL reviewed your changes and applied them ... you can proceed

**Comment by manniL** [comment]
Thank you ☺️

**Pull request merged**